lib/client.js

```diff
@@ -5,6 +5,7 @@
  * Dependencies
  */
 
```

Comment:
exposing the private methods/properties is good because it allows us to unit test, see: https://github.com/millermedeiros/threads/commit/843425d5e0a2a64f2eb1794bf7e0eefd3758af5c#diff-3d6cd7d35d59ce33f7b9c33ff2d59a20R20